Task 2/Task 2.1.2/Task 2.1.2/Program.cs

```diff
@@ -0,0 +1,804 @@
+                    Console.WriteLine("Такой команды не существует! Введите другую команду.");
+                    CustomPaint(name, figures);
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Метод, использующися для ввода координат точки фигуры. Например, центр курга, конечной точки линии или вершины треугольника.
+        /// </summary>
+        /// <returns>Возвращает float значение координаты</returns>
+        static float InputCoordinats ()
+        {
+            Console.WriteLine("Введите координату (только число):");
+            do
+            {
+                string usercoordinate = Console.ReadLine();
+                if (float.TryParse(usercoordinate, out float coordinate))
+                    return coordinate;
+            }
+            while (true);
+        }
+
+        /// <summary>
+        /// Метод, использующися для проверки введеных значений.
+        /// </summary>
+        /// <returns>Возвращает удовлетворяющее условиям проверки float значение</returns>
+        static float CheckingValues (float checkvalue)
+        {
+            string uservalue = Console.ReadLine();
+            float.TryParse(uservalue, out float value);
+            while (value <= checkvalue)
+            {
+                Console.WriteLine("Вы ввели некорректное значение. Попробуйте еще раз");
+                uservalue = Console.ReadLine();
+                float.TryParse(uservalue, out value);
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Метод, использующися для проверки существования треугольника. 
+        /// Сумма двух меньших сторон треугольника должна быть больше или равна величине его большей стороны.
+        /// </summary>
+        static void IsTriangleValid(Triangle triangle)
+        {
+            if (triangle.SideA >= triangle.SideB && triangle.SideA >= triangle.SideC && triangle.SideA <= triangle.SideB + triangle.SideC)
+            {
+                Console.WriteLine("Треугольник валиден.");
+
+            }
+            else if (triangle.SideB >= triangle.SideA && triangle.SideB >= triangle.SideC && triangle.SideB <= triangle.SideA + triangle.SideC)
+            {
+                Console.WriteLine("Треугольник валиден.");
+
+            }
+            else if (triangle.SideC >= triangle.SideA && triangle.SideC >= triangle.SideB && triangle.SideC <= triangle.SideA + triangle.SideB)
+            {
+                Console.WriteLine("Треугольник валиден.");
+            }
+            else
+            {
+                Console.WriteLine("Такого треугольника не существует, придется вам переввести значения его сторон.");
+                triangle.SideA = CheckingValues(0);
+                triangle.SideB = CheckingValues(0);
+                triangle.SideC = CheckingValues(0);
+                Console.WriteLine("И теперь еще раз пройти проверку на валидность.");
+                IsTriangleValid(triangle);
+            }
+        }
```

Comment:
While asking and printing stuff is okay to do outside triangle (and in most cases IS the way), checking if triangle is valid is triangle's work. Classes must make sure that they are valid, they don't depend on others like that.